### DISCUSSION TITLE
poiseuille flow

### DISCUSSION THREAD

**Discussion by phy18d003** [pypr/pysph#369] [General]
I would like to know what is the constant ''alpha''  taken for artificial viscosity term (or how it is defined) in the poiseuille flow example solved in PySPH.  Please help me on how to  print values of such constants in the code.

**Comment by abhinavmuta** [comment]
Hi,

The [Poiseuille flow example](https://github.com/pypr/pysph/blob/master/pysph/examples/poiseuille.py) is not using the artificial viscosity equation. The flow is viscous with a kinematic viscosity matching the Reynolds number of [0.0125](https://github.com/pypr/pysph/blob/master/pysph/examples/poiseuille.py#L35). When you run a simulation, all the equations and their parameter values are printed to the log file ending with a ".log" extension in the output directory.
